setup.py

```diff
@@ -26,6 +26,7 @@
         "rlp>=0.4.7,<1.0.0",
         "eth-keys>=0.2.0b3,<1.0.0",
         "trie>=1.3.2,<2.0.0",
+        "pylru>=1.0.9",
```

Comment:
It looks like https://pypi.python.org/pypi/lru-dict might be worth evaluating as it's implemented in C.

Reply:
For this limited one-block-import test I'm running it doesn't seem to provide any benefits. I guess it depends how large this cache can get. Maybe just leave a comment there to evaluate it properly if `_get_account()` shows up again as one of the worst offenders in cProfile stats?

Reply:
:-1: `pylru` is GPL https://github.com/jlhutch/pylru/blob/master/LICENSE.txt

Reply:
Should we go with lru-dict, then?

Reply:
@gsalgado I think yes, (that's what web3.py is using at least).